src/fparser/common/utils.py

```diff
@@ -81,6 +82,7 @@
            'CHAR_BIT','str2stmt',
            'classes']
 
+import logging
```

Comment:
Although not strictly part of this PR, pylint complains that the `import sys` below is not needed.

Reply:
OK, I've fixed this (although there are many other pylint errors not related to this change in this file).